packages/web3-core/src/web3_request_manager.ts

```diff
@@ -155,7 +167,17 @@ export class Web3RequestManager<
 		Method extends Web3APIMethod<API>,
 		ResponseType = Web3APIReturnType<API, Method>,
 	>(request: Web3APIRequest<API, Method>): Promise<ResponseType> {
-		const response = await this._sendRequest<Method, ResponseType>(request);
+
+		let requestObj = {...request};
+
+		if (!isNullish(this.middleware))
```

Comment:
it is better to have brackets `{` and `}` even if it is very simple `if`. because in the future when someone extends this code they can make a mistake